docs/content/docs/tasks_usage/observability/_index.md

```diff
@@ -10,6 +10,6 @@ aliases: ["observability"]
 OSM's observability stack includes Prometheus for metrics collection, Grafana for metrics visualization, Jaeger for tracing and Fluent Bit for log forwarding to a user-defined endpoint. These are disabled by default but may be enabled during OSM installation with flags `--deploy-prometheus`, `--deploy-grafana`, `--deploy-jaeger` and `--enable-fluentbit`.
 
 ## Table of Contents
-- [Metrics - Prometheus and Grafana](../../tasks_usage/metrics.md)
+- [Metrics - Prometheus and Grafana](./metrics.md)
 - [Tracing - Jaeger](./tracing.md)
-- [Log forwarding - Fluent Bit](../../tasks_usage/logs.md)
+- [Log forwarding - Fluent Bit](./logs.md)
```

Comment:
Could you verify this renders on the website correctly.

Reply:
This can work on the website, the format followed the internal-links like:

```
// 1
---
title: "Foo.md"
description: "Foo"
type: docs
aliases: ["logs.md"]
---

Here's a link to [Log](./logs.md).
```